### PR TITLE
Add Cancel/Replace mechanism on new order command

### PR DESCRIPTION
**Pull request by alexppxela**
Signed-off-by: Alexandre Thibault <alexandre.thibault@artex.io>

### CODE REVIEW

File: cmd/new/order/order.go [hunk 1]
```diff
@@ -247,6 +273,10 @@ LOOP:
 				return err
 			}
 
+			if msgType, err := msg.Header.GetString(tag.MsgType); err == nil && msgType == "8" {
```
Comment (sylr) [inline cmd/new/order/order.go:276]:
```suggestion
			if msgType, err := msg.Header.GetString(tag.MsgType); err == nil && msgType == enum.MsgType_EXECUTION_REPORT {
```

File: cmd/new/order/order.go [hunk 2]
```diff
@@ -73,6 +76,10 @@ func init() {
 
 	NewOrderCmd.Flags().BoolVar(&optionStopOnFinalState, "stop-on-final-state", false, "Stop application when receiving an order with a final state")
 
+	NewOrderCmd.Flags().DurationVar(&optionUpdatePeriod, "update-period", 0*time.Second, "Period for recurring update order price/quantity")
```
Comment (sylr) [inline cmd/new/order/order.go:79]:
```suggestion
	NewOrderCmd.Flags().DurationVar(&optionUpdatePeriod, "update-period", 0*time.Second, "Period for recurring order price/quantity updates")
```

File: cmd/new/order/order.go (hunk 1 above)
Comment (alexppxela) [reply cmd/new/order/order.go]: done

File: cmd/new/order/order.go (hunk 2 above)
Comment (alexppxela) [reply cmd/new/order/order.go]: done

**Pull request merged**